src/main/java/FileUtils/CSVReader.java

```diff
@@ -0,0 +1,28 @@
+package FileUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Created by p998tbd on 2016.12.13.
+ */
+public class CSVReader implements DecathlonDataReader {
+
+    public List<List<String>> loadFile(String fileName) throws IOException {
+        try {
+            File file = new File(fileName);
+            Stream<String> lines = Files.lines(file.toPath());
+            List<List<String>> values = lines.map(line -> Arrays.asList(line.split(";")))
```

Comment:
No need for extra variable you can just:
`return lines.map(line -> Arrays.asList(line.split(";"))).collect(Collectors.toList());`